.github/workflows/build-node-bridge.yml

```diff
@@ -0,0 +1,49 @@
+name: "[Build] suite-node-bridge-bin-js"
```

Comment:
if you search for 

```
yarn workspace @trezor/transport-bridge build:lib
```
there are a couple of occurrences. and this very script actually calls 
```
yarn workspace @trezor/transport-bridge build:js
```

not sure how good idea this is but it could be feasible just to reuse artifacts from another job and spare us one yml config file. for example https://github.com/trezor/trezor-suite/blob/develop/.github/workflows/test-suite-desktop-e2e.yml#L1 could work.